app/controllers/service_calendars_controller.rb

```diff
@@ -73,6 +73,7 @@ def update
   end
 
   def table
+    initialize_service_request
```

Comment:
The reason I added the procs to the before filters was that this was causing an issue before with authorization errors in the Dashboard. Instead, you can assign @service_request in the before_filter "authorize_dashboard_access", or more specifically I think, "authorize_protocol"